Editing and Formatting for Phase 2 (#279)
Technical-writing-mentorship-program/Technicalwritingcourse: docs/technical-writing-course/Editing.mdx

Everyone can write, but what makes a valuable document is its conciseness and correctness. Your documentation must be edited to fit all required criteria Such as style, tone, audience needs, and more.

:::tip
 Robert Cormier says - “The beautiful part of writing is that you don't have to get it right the first time, unlike, say, a brain surgeon. You can always do it better; find the exact word, the apt phrase, the leaping simile."
:::

Editing is the layer that makes your technical documentation better.

:::note
In this Module, you will learn:
:::

- The importance of editing,
- The different types of editing, and
- How to edit for:
  - Clarity
  - Conciseness
  - Correctness

## Benefits of editing

Editing is an essential step in the documentation process for several reasons. It ensures clarity, conciseness, and accuracy in a technical document. We will explore this aspect in details in the coming sections of this module.

**_Other benefits of editing:_**

> 1. ###### Error Prevention and Correction

Reviewer: ```suggestion
1. ### Error Prevention and Correction
```

Editing ensures meticulous proofreading to identify and correct errors in grammar, spelling, punctuation, and technical details. Documentation must be error-free to maintain credibility.

> 2. ###### Consistency in Style and Terminology

Reviewer: ```suggestion
2. ### Consistency in Style and Terminology
```

Editing ensures consistent terminology, style, and formatting throughout the document. Consistent language helps readers understand and interpret technical information, reducing the risk of confusion or misinterpretation.

> 3. ###### Alignment with Standards and Guidelines

Reviewer: ```suggestion
3. ### Alignment with Standards and Guidelines
```

Organizations have specific standards and guidelines for technical documentation. Editing ensures the document aligns with these standards. It aids compliance and adherence to the organizations' regulations.

> 4. ###### Enhancement of Readability and Accessibility

Reviewer: ```suggestion
4. ### Enhancement of Readability and Accessibility
```

Editing involves breaking down complex ideas into digestible pieces, using appropriate headings and subheadings, and incorporating visual elements. Enhanced readability makes technical content more user-friendly for a diverse audience.

> 5. ###### Logical Organization of Information

Reviewer: ```suggestion
5. ### Logical Organization of Information
```

:::

> It's important to note that proofreading needs a keen eye for detail, a strong command of language, and a thorough understanding of the subject matter. 

Reviewer: ```suggestion
It is important to note that proofreading needs a keen eye for detail, a strong command of language, and a thorough understanding of the subject matter. 
```

> ### Copy Editing

Reviewer: ```suggestion
### Copy Editing
```

<!-- ![How to editing](/assets/images/How-to-editing-CCC.jpg) -->

> <img src="https://dev-to-uploads.s3.amazonaws.com/uploads/articles/xykfc692ilw3uf886sdu.jpg" 

Reviewer: - Remove hosted image links
- Add images to the `img` folder
- Add figure caption

Editing improves the quality of your technical documentation, making it a valuable resource that communicates concisely and accurately to your target audience.

> ### Exercise

Reviewer: Remove heading and add the tip callout
